operator/cmd/mesh/testdata/manifest-generate/data-snapshot/addons/gen.sh

```diff
@@ -70,7 +70,6 @@ helm3 template prometheus stable/prometheus \
   kubectl create configmap -n istio-system istio-grafana-dashboards \
```

Comment:
Ignore everything in `data-snapshot` - its just synced as the tests fail against stale test data